examples/style_sheets/style_sheets_reference.py

```diff
@@ -25,8 +24,8 @@ def plot_colored_sinusoidal_lines(ax):
     """
```

Comment:
Wouldn't “Plot sinusoidal lines with colors following the style color cycle.” be more correct?